### PR TITLE
Remove extra margin between soft buttons at small widths

### PR DESCRIPTION
**Pull request by joshlory**
Keeps the buttons all on the same line when resizing.

Before:

compact-buttons-before

After:

compact-buttons-after

### CODE REVIEW

**Comment by cpirich** [comment]
I think `resizeVisualization` is only called when the user manually moves the grippy. Not when the browser resizes. I think you want to also add this style info to the existing responsive classes with the same breakpoints. In this section of common.scss:

```
@media screen and (max-width: 1000px) {
  $width: 200px;
```

**Comment by joshlory** [comment]
Good catch, fixed in 01f9ea964daabbb5b1f87eaffcc14806468bc63f.

File: apps/src/StudioApp.js [hunk 1]
```diff
@@ -1111,6 +1112,12 @@ StudioApp.prototype.resizeVisualization = function (width) {
     visualizationEditor.style.marginLeft = newVizWidthString;
   }
 
+  if (oldVizWidth < 230 && newVizWidth >= 230) {
```
Comment (Bjvanminnen) [inline apps/src/StudioApp.js:1115]: Why is 230 the magic number? Worth a comment?

**Comment by Bjvanminnen** [comment]
lgtm

**Pull request merged**